commons/zenoh-util/src/std_only/net/mod.rs

```diff
@@ -210,12 +210,19 @@ pub fn get_multicast_interfaces() -> Vec<IpAddr> {
     }
 }
 
-pub fn get_local_addresses() -> ZResult<Vec<IpAddr>> {
+pub fn get_local_addresses(interface: Option<String>) -> ZResult<Vec<IpAddr>> {
```

Comment:
It seems this function does not need to take ownership of the `String`, maybe pass `Option<&str>`?